ReactiveXComponent/RabbitMq/XComponentException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace ReactiveXComponent.RabbitMq
+{
+    [Serializable]
+    public class XComponentException : Exception
```

Comment:
You should rename the class to _ReactiveXComponentException_. _XComponentException_ seems to refer to a core exception in XComponent.